lib/command/config.rb

```diff
@@ -34,11 +34,11 @@ def cache_path
     end
 
     def prebuild_sandbox_path
-      @dsl_config[:prebuild_sandbox_path] || @deprecated_config["prebuild_path"] || "_Prebuild"
+      @dsl_config[:prebuild_sandbox_path] || @deprecated_config["prebuild_path"] || "#{Dir.pwd}/_Prebuild"
     end
 
     def prebuild_delta_path
-      @dsl_config[:prebuild_delta_path] || @deprecated_config["prebuild_delta_path"] || "_Prebuild_delta/changes.json"
+      @dsl_config[:prebuild_delta_path] || @deprecated_config["prebuild_delta_path"] || "#{Dir.pwd}/_Prebuild_delta/changes.json"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/grab/cocoapods-binary-cache/blob/89bcaa56b63e6a96f7f3da7770eb000570ad4cb2/lib/command/config.rb#L41"></span>Layout/LineLength: Line is too long. [129/120]</td>
    </tr>
  </tbody>
</table>